auth.go

```diff
@@ -27,7 +27,7 @@ func (api *Client) SendAuthRevoke(token string) (*AuthRevokeResponse, error) {
 	return api.SendAuthRevokeContext(context.Background(), token)
 }
 
-// SendAuthRevokeContext will retrieve the satus from api.test
+// SendAuthRevokeContext will retrieve the satus from api.revoke
```

Comment:
I think this comment is wrong in itself... 🤔
The `SendAuthRevoke` method sends a revocation request for the token.

Reply:
Sorry, missed my local change :P I have pushed an updated version.